roles/runtime/tasks/setup_aws.yml

```diff
@@ -60,4 +60,82 @@
       register: __dw_builds_async
       until: __dw_builds_async.finished
       retries: 120
-      delay: 30
\ No newline at end of file
+      delay: 30
+
+    - name: Retrieve CDP DW experiences
+      cloudera.cloud.dw_cluster_info:
+        env: "{{ run__env_name }}"
+      register: run__dw_list
+
+    - name: Create CDP DW Database catalogs
+      when: not __dw_dbc_config.use_default_dbc
+      cloudera.cloud.dw_database_catalog:
+        cluster_id : "{{ run__dw_list.clusters[0].id }}"
```

Comment:
Perhaps pull out into a separate task that can ensure the cluster 0 exists, etc. -- the `set_fact` variable could be `__default_dbc` etc.